Node/avg.js

```diff
@@ -0,0 +1,25 @@
+function calcAvgNum(num1, num2, num3){
```

Comment:
Currently, the function `calcAvgNum` only handles 3 arguments. To improve flexibility, refactor the function to accept an array of numbers, allowing the average to be calculated for any number of inputs. This would make the function more reusable across different scenarios.

Reply:
I have now updated the function with the recommended changes